cantusdb/turtle/csv2rdf_single_subject.py

```diff
@@ -0,0 +1,71 @@
+import csv
+import validators
+import sys
+import json
+from rdflib import Graph, URIRef, Literal
+from rdflib.namespace import RDF
+from typing import Optional
+
+# a main operation function
+def convert_csv_to_turtle(filename) -> Graph:
+    """
+    Adds all informations as RDF triples from the input filename into a graph and return it.
+    @Pre: input filename must be type string.
+    @Post: Returns a RDF.Graph that has all the triples
+    """
+
+    if len(sys.argv) != 2:
+        raise ValueError("Invalid number of input filename")
+
+    with open(filename, "r", encoding="utf-8") as csv_file:
+        # TODO: maybe change this one to accomadate for multiple files
+
+        g = Graph()
+
+        csv_reader = csv.reader(csv_file)
+        with open("relations_mapping.json", "r") as mapper:
+            ontology_dict = json.load(mapper)
+
+        header = next(csv_reader)
+        header = header[1:]
```

Comment:
A comment or descriptive variable name would be helpful then. So what about:
`header_without_subject = header[1:]` or
```
# Here we remove the first value in the header since the first column contains the subject
header = header[1:]
```

Reply:
Ok, I will fix it.